pkg/terminal/terminal.go

```diff
@@ -210,22 +211,36 @@ func (t *Term) Run() (int, error) {
 	signal.Notify(ch, syscall.SIGINT)
 	go t.sigintGuard(ch, multiClient)
 
+	fns := trie.New()
+	cmds := trie.New()
+	funcs, _ := t.client.ListFunctions("")
+	for _, fn := range funcs {
+		fns.Add(fn, nil)
+	}
+	for _, cmd := range t.cmds.cmds {
+		for _, alias := range cmd.aliases {
+			cmds.Add(alias, nil)
+		}
+	}
+
 	t.line.SetCompleter(func(line string) (c []string) {
+		prefix := ""
 		if strings.HasPrefix(line, "break ") || strings.HasPrefix(line, "b ") {
```

Comment:
We should also check for `trace`, it didn't do it before but clearly it's wrong.

Reply:
Done.